lib/ddtrace/span.rb

```diff
@@ -402,10 +378,12 @@ def started?
       !@start_time.nil?
     end
 
-    # Return whether the duration is finished or not.
-    def finished?
+    # Return whether the duration is stopped or not.
+    def stopped?
       !@end_time.nil?
     end
+    # DEPRECATED: Use #stopped? instead
```

Comment:
We can document this deprecation in a YARD-compatible way, like so: https://www.rubydoc.info/gems/yard/file/docs/Tags.md#deprecated